src/TraceStream.cc

```diff
@@ -573,7 +573,7 @@ TraceFrame TraceReader::read_frame(FrameTime skip_before) {
       const auto& hole = holes[j];
       h[j] = { hole.getOffset(), hole.getSize() };
     }
-    raw_recs[i] = { w.getAddr(), (size_t)w.getSize(), i32_to_tid(w.getTid()), h };
+    raw_recs[i] = { w.getAddr(), (size_t)w.getSize(), i32_to_tid(w.getTid()), h, w.getSizeIsConservative()};
```

Comment:
Somewhere there should be a call to `setSizeIsConservative()` but I think it's missing.

Reply:
Uh, whoops, yeah, this is very obviously not going to do anything useful without it.